google/auth/crypt/es.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2017 Google Inc.
```

Comment:
Note to self (and other reviewers): most of this file is copied over from `es256.py`, so I only focused on the changes there